src/Test.UnitTests.Sarif/Core/LocationTests.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Core
     public class LocationTests
     {
         private const string Id = "\"Id\"";
+        private static readonly TestAssetResourceExtractor Extractor = new TestAssetResourceExtractor(typeof(LocationTests));
```

Comment:
>Extractor [](http://example.com/codeflow?start=59&length=9)

the other file has:
private static readonly IResultMatcher s_matcher = new V2ResultMatcher();
should we renaming to have s_ similarly

Reply:
Scrubbed for this.